python/tk_multi_publish2/dialog.py

```diff
@@ -792,6 +833,10 @@ def _synchronize_tree(self):
             # drop area
             self._progress_handler.progress_details.set_parent(
                 self.ui.large_drop_area)
+
+            if not self.manual_load_enabled:
+                self._error_overlay.show_no_items_error()
```

Comment:
I'm still surprised that this doesn't show up on top of the frame, given its parent.

Reply:
I also tried, using same parent as a different instance, same result.